src/main/java/edu/harvard/iq/dataverse/ProvUtilFragmentBean.java

```diff
@@ -94,6 +101,370 @@ public String getPrettyJsonString(JsonObject jsonObject) {
         Gson gson = new GsonBuilder().setPrettyPrinting().create();
         return gson.toJson(je);
     }
+
+    public boolean isProvValid(String jsonInput) {
+        try { 
+            schema.validate(new JSONObject(jsonInput)); // throws a ValidationException if this object is invalid
+        } catch (ValidationException vx) {
+            logger.info("Prov schema error : " + vx); //without classLoader is blows up in actual deployment
+            return false;
+        } 
+
+        return true;
+    }
     
-   
+    //Pulled from https://www.w3.org/Submission/2013/SUBM-prov-json-20130424/schema
+    //Not the prettiest way of accessing the schema, but loading the .json file as an external resource
+    //turned out to be very painful, especially when also trying to exercise it via unit tests
+    public final String provSchema = 
+        "{\n" +
```

Comment:
You should probably look at how RemoteDataFrameService loads a file in #4814 

        InputStream resourceStream = RemoteDataFrameService.class.getResourceAsStream(path);
        String resourceAsString = "";

        // Try opening a buffered reader stream
        try {
            resourceAsString = IOUtils.toString(resourceStream, "UTF-8");
            resourceStream.close();
        } catch (IOException ex) {
            logger.warning(String.format("RDATAFileReader: (readLocalResource) resource stream from path \"%s\" was invalid", path));
        }
        return resourceAsString;`